lib/wickedbase.pm

```diff
@@ -457,6 +457,52 @@ sub validate_macvtap {
     validate_script_output("cat $macvtap_log", sub { m/Success listening to tap device/ });
 }
 
+sub setup_bond() {
+    my ($self, $mode, $iface0, $iface1) = @_;
+    my $ifsut = $mode == "balance-rr" ? "bond0-rr"
```

Comment:
maybe it would be easier just to pass needed string and having this huge trinary pyramid ?

Reply:
ah you already did it "right" for teaming . so what the problem with bonding ? why not do the same ?

Reply:
I was just following same code as in legacy test suite:
https://gitlab.suse.de/wicked-maintainers/wicked-testsuite/blob/stable/features/step_definitions/wicked_actions.rb#L2415
(see that it doesn't do it for team test).

But sure, the test can pass directly this value instead of translating it.